src/mame/drivers/starshp1.cpp

```diff
@@ -197,7 +197,10 @@ static INPUT_PORTS_START( starshp1 )
 	PORT_DIPNAME( 0x20, 0x20, "Extended Play" )
 	PORT_DIPSETTING(    0x00, DEF_STR( No ) )
 	PORT_DIPSETTING(    0x20, DEF_STR( Yes ) )
-	PORT_BIT( 0x40, IP_ACTIVE_LOW, IPT_BUTTON3 ) PORT_TOGGLE
+	/* IPT_BUTTON3 is the Speed lever (Throttle)                */
+	/* This is _not_ IPT_TOGGLE, even though it looks like one. */
+	/* It returns to SLOW unless you hold it down (FAST)        */
+	PORT_BIT( 0x40, IP_ACTIVE_LOW, IPT_BUTTON3 )
```

Comment:
Can you please either use C++ line comments, or a single C block comment?  Doing it like this gets the worst of both worlds - hard to reformat, visual clutter at the beginning and end of each line, and stuff that needs to be realigned if lines change length.